drivers/storage/portworx/components.go

```diff
@@ -344,6 +366,107 @@ func createVolumePlacementStrategyCRD() error {
 	return k8s.Instance().ValidateCRD(resource, 1*time.Minute, 5*time.Second)
 }
 
+func createCSINodeInfoCRD() error {
```

Comment:
Currently we don't manually create a CSINodeInfo CRD. Our CSINodeInfo object is created by the Node Driver Registrar:

https://kubernetes-csi.github.io/docs/node-driver-registrar.html

This sidecar is in the Portworx daemonset and continually registers the CSINodeInfo depending on the status of that PX node.